packages/core/src/order/order-request-sender.spec.ts

```diff
@@ -180,6 +181,31 @@ describe('OrderRequestSender', () => {
             }
         });
 
+        it('throws `InvalidShippingAddressError` if error type is `invalid_shipping_address`', async () => {
+            const error = getErrorResponse(
+                {
+                    status: 400,
+                    title: 'Invalid shipping address',
+                    type: 'invalid_shipping_address',
+                },
+                undefined,
+                409,
+            );
+
+            jest.spyOn(requestSender, 'post').mockReturnValue(Promise.reject(error));
+
+            try {
+                const payload = {
+                    cartId: 'b20deef40f9699e48671bbc3fef6ca44dc80e3c7',
+                    useStoreCredit: false,
+                };
+
+                await orderRequestSender.submitOrder(payload);
+            } catch (error) {
+                expect(error).toBeInstanceOf(InvalidShippingAddressError);
+            }
```

Comment:
Thought we typcially avoid catching error in a test.
Perhaps something like this.
```
await expect(orderRequestSender.submitOrder(payload)).rejects.toThrow(InvalidShippingAddressError);
```

Reply:
Let me try that, I just copied the previous test 😄

Reply:
Updated both tests.